### PR TITLE
Check task was set before accessing the cluster

### LINKED ISSUE #250: Improve UX When Dealing With Tasks
The current functionality does not encourage nor enforces the use of start_task/end_task commands.

Given the commands are fundamental for the gamified experience, this should be addressed.

### PR DESCRIPTION
**Pull request by s-irvine**
Closes #250

### CODE REVIEW

File: attack/scripts/bashrc [hunk 1]
```diff
@@ -86,6 +86,21 @@ find_current_task() {
 }
 readonly -f find_current_task
 
+check_if_task_set() {
+  local task
+  task=$(jq -r '.current_task' /progress.json)
+  if [[ ${task} = "null" ]]; then
+    echo "You have not started a task yet. One can be set with 'start_task ...'. Would you like to continue anyway without starting a task? Enter '1' or '2'"
```
Comment (jondkent) [inline attack/scripts/bashrc:93]: I think you mean enter yes/no as that is what the following case statements looks for

**Review by jondkent — CHANGES_REQUESTED** [review]

File: attack/scripts/bashrc (hunk 1 above)
Comment (abdullahgarcia) [reply attack/scripts/bashrc:93]: +1

**Review by abdullahgarcia — COMMENTED** [review]
Same comment as @jondkent .

File: attack/scripts/bashrc (hunk 1 above)
Comment (s-irvine) [reply attack/scripts/bashrc:93]: This is using the `select` command and it gives the user an option like this:

```
root@attack:/attack[0]$ starting_point 
You have not started a task yet. One can be set with 'start_task ...'. Would you like to continue anyway without starting a task? Enter '1' or '2'
1) Yes
2) No
#? 
```
So you need to actually enter a number. If you don't it won't work:
```
root@attack:/attack[1]$ starting_point 
You have not started a task yet. One can be set with 'start_task ...'. Would you like to continue anyway without starting a task? Enter '1' or '2'
1) Yes
2) No
#? Yes
#? No
#? y
#? yes
#? no
#? 2
root@attack:/attack[1]$
```

**Review by jondkent — APPROVED** [review]
LGTM

**Pull request merged**